src/cli.ts

```diff
@@ -32,14 +41,19 @@ export function run(): Promise<void> {
     }
 
     if (cliOptions.help) {
-      var usage = cli.getUsage({
-        header: 'A development server for Polymer projects',
-        title: 'polyserve',
-      });
-      console.log(usage);
+      printUsage(cli);
+      reject();
```

Comment:
I'd rather resolve than reject(). Rejecting a Promise is like throwing an exception, but this isn't an exceptional case, it's expected and valid user input.